MachObfuscator/SymbolMangling/CezarMangler/CaesarCypher.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+final class CaesarCypher {
+    private let asciiRange = UInt8(33) ... UInt8(126)
+
+    func cypher(element: UInt8, cypherKey: UInt8) -> UInt8 {
```

Comment:
What do you think about using some verb?
```suggestion
    func encrypt(element: UInt8, key: UInt8) -> UInt8 {
```

Reply:
hmm there is such verb as `to cypher`. actually, I like both encrypt and cypher, maybe encrypt would be even better visually..? let me know what you think